cmd/mt_receive_adapter/main.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2020 The Knative Authors
+Copyright 2020-2021 The Knative Authors
```

Comment:
I think we just do 2021 ? Not exactly sure, but that's what I've noticed for 2020 (got just updated to that from 2019)

Reply:
I have never found any resource that explains how to deal with dates in copyright headers and it always puzzles me 😄
Nothing against just using `2021` on my side.

Reply:
@matzew Done in [f229ef26](https://github.com/knative-sandbox/eventing-github/compare/22d4d05f0ed8044083aad4cd945075caeef31b9e..f229ef265160ecc5a04ab8e1595600d6f6ff7f60) sir